Sources/KlaviyoCore/Utils/FileUtils.swift

```diff
@@ -69,27 +69,25 @@ public func removeFile(at url: URL) -> Bool {
     return false
 }
 
-/// load any plist from app main bundle or React Native framework bundle
+/// Load plist from main application bundle
 /// - Parameter name: the name of the plist
-/// - Returns: the contents of the plist in `[String: AnyObject]` or nil if not found
+/// - Returns: the contents of the plist or nil if not found
 func loadPlist(named name: String) -> [String: AnyObject]? {
```

Comment:
Making this method general purpose and moving RN specific logic to a specialized method.